crispy_flame/lib/components/background.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:crispy_flame/crispy_flame.dart';
+import 'package:flame/components/parallax_component.dart';
+import 'package:flutter/cupertino.dart';
+
+class Background {
+  BoxGame game;
+  Paint bgPaint;
+  ParallaxComponent parallaxComponent;
+  Rect bgRect;
+
+  Background(this.game) {
+    parallaxComponent = ParallaxComponent();
+    parallaxComponent.load(["background.jpg"]);
+    game.add(parallaxComponent);
```

Comment:
adding to game should be done outside this scope. If you extend `Component` you might easily track this outside